Harubee-iOS/Sources/Presentation/Onboarding/NavigationHeaderView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NavigationHeaderView.swift
+//  Harubee-iOS
+//
+//  Created by Seo-Jooyoung on 11/2/24.
+//  Copyright © 2024 namudiEtc. All rights reserved.
+//
+
+import SwiftUI
+
+enum PageNumber {
+  case first, second, third
+  
+  var pageNumber: Int {
+    switch self {
+    case .first:
+      return 1
+    case .second:
+      return 2
+    case .third:
+      return 3
+    }
+  }
+}
+
+struct NavigationHeaderView: View {
```

Comment:
온보딩에서 사용되는 NavigationHeader여서 다음과 같이 수정해보았습니다.
```suggestion
struct OnboardingNavigationHeaderView: View {
```

Reply:
좋은 것 같아요 ! 수정했습니당